application/controllers/PluginsController.php

```diff
@@ -7,143 +7,6 @@
 class PluginsController extends LSYii_Controller
 {
 
-    //public $layout = 'main';
-
-    /**
-     * Stored dynamic properties set and unset via __get and __set.
-     * @var array of mixed.
-     */
-    protected $properties = array();
-
-    public function __get($property)
-    {
-        return $this->properties[$property];
-    }
-
-    public function __set($property, $value)
-    {
-        $this->properties[$property] = $value;
-    }
-
-    public function _init()
-    {
-        parent::_init();
-        Yii::app()->bootstrap->init(); // Make sure bootstrap css is rendered in time
-    }
-
-    /**
-     * Activates plugin with $id
-     *
-     * @param int $id
-     * @return void
-     */
-    public function actionActivate($id)
```

Comment:
Ore leave it and redirect to admin controller ? I never use it in plugins (and it's not in API)

Reply:
I say remove it. It has no place here.